### ISSUE TITLE
RtImages.java:96-100: Several API calls required an...

### ISSUE THREAD

**Issue by 0pdd** [amihaiemil/docker-java-api#99]
The puzzle `83-127c3a0c` from #83 has to be resolved:

https://github.com/amihaiemil/docker-java-api/blob/bc8e27a93cff11821587f082de9b238e0a506af6/src/main/java/com/amihaiemil/docker/RtImages.java#L96-L100

The puzzle was created by Mihai A on 07-May-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/G6LPQQV2P) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#99](https://github.com/amihaiemil/docker-java-api/issues/99) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#99](https://github.com/amihaiemil/docker-java-api/issues/99) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1301355019); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on review/merge of #110

**Comment by 0crat** [comment]
> @0crat waiting on review/merge of #110 ([here](https://github.com/amihaiemil/docker-java-api/issues/99#issuecomment-389310826))

@llorllale The impediment for [#99](https://github.com/amihaiemil/docker-java-api/issues/99) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@6a151509-d0d4-40fa-a569-53d8cdf9ca67, thread: AsyncFlush-6
com.zerocracy.farm.reactive.StkGroovy[125] java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/in/orders/close_job.groovy
sun.reflect.NativeMethodAccessorImpl[-2] java.lang.reflect.InvocationTargetException: null
com.amazonaws.http.AmazonHttpClient$RequestExecutor[1113] com.amazonaws.SdkClientException: Unable to execute HTTP request: null
sun.nio.ch.FileChannelImpl[110] java.nio.channels.ClosedChannelException: null
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): CID: [1471237887](https://www.0crat.com/G6LPQQV2P/1471237887), Type: "Close job", Author: "amihaiemil"

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#99](https://github.com/amihaiemil/docker-java-api/issues/99) is now out of scope
